integrations/apache-camel/src/main/java/org/apache/plc4x/camel/Plc4XConsumer.java

```diff
@@ -112,25 +114,25 @@ private PlcSubscriber getSubscriber() {
     }
 
     @Override
-    public void accept(SubscriptionEventItem subscriptionEventItem) {
-        LOGGER.debug("Received {}", subscriptionEventItem);
+    public void accept(PlcSubscriptionEvent plcSubscriptionEvent) {
+        LOGGER.debug("Received {}", plcSubscriptionEvent);
         try {
             Exchange exchange = endpoint.createExchange();
-            exchange.getIn().setBody(unwrapIfSingle(subscriptionEventItem.getValues()));
+            exchange.getIn().setBody(unwrapIfSingle(plcSubscriptionEvent.getAllObjects("default")));
```

Comment:
where does this "default" constant come from?

Reply:
I have to give everything a name ... so I just chose one ...